docs/cunumeric/source/comparison/_comparison_generator.py

```diff
@@ -30,7 +30,11 @@ def _filter(obj, n):
                 "set_numeric_ops",
                 "size",
                 "sometrue",
-            ]  # not in blacklist
+                "loads",
+                "mafromtxt",
+                "matmul",
+                "ndfromtxt",
```

Comment:
Should these exist/be copied over? Its possible they do get copied over in #219 but I have not checked yet, but they are not currently being copied on `branch-22.03`

Reply:
I think they should.

Reply:
I think they should.

Reply:
OK, since they are already previously omitted, can we fix the omission in a dedicated PR that also reverts this comparison blocklist change? Fixing the historical omission does not seem directly related to the immediate docs build issue.

Reply:
Yes, that sounds like a right thing to do.

Reply:
Awesome, submitted https://github.com/nv-legate/cunumeric/issues/226